test/jobs/send_daily_triage_email_job_test.rb

```diff
@@ -8,20 +8,64 @@ def setup
 
   test 'send out daily triage email' do
     def @user.issue_assignments_to_deliver; IssueAssignment.all.limit(1); end
+    def @job.before_email_time_of_day?(*); false; end
 
     assert_kind_of Mail::Message, @job.perform(@user)
   end
 
   test 'does not deliver if no subscriptions' do
     def @user.issue_assignments_to_deliver; []; end
+    def @job.before_email_time_of_day?(*); false; end
 
     refute @job.perform(@user)
   end
 
   test 'does not deliver if email should be skipped' do
     def @user.issue_assignments_to_deliver; IssueAssignment.all.limit(1); end
+    def @job.before_email_time_of_day?(*); false; end
     def @job.skip_daily_email?(*); true; end
 
     refute @job.perform(@user)
   end
+
+  test 'when email_time_of_day not set, it delivers after default time of day' do
+    def @user.issue_assignments_to_deliver; IssueAssignment.all.limit(1); end
+
+    Timecop.freeze time_preference_for_today(SendDailyTriageEmailJob::DEFAULT_EMAIL_TIME_OF_DAY) + 1.hour do
```

Comment:
Rather than use a new testing library, we can just use [Minitest stubs](https://github.com/seattlerb/minitest/blob/master/lib/minitest/mock.rb#L212).

Time.stub(:now, your_time) do ...